### PR TITLE
updates for u12

### PR DESCRIPTION
**Pull request by mbcoder**
Prep for upcoming release

### CODE REVIEW

**Review by Rachael-E — COMMENTED** [review]
thanks @mbcoder , a couple of comments to consider this time round. we should also update the gradle wrapper properties to point to 7.0.2 version of gradle to be consistent with samples.

File: build.gradle [hunk 1]
```diff
@@ -58,3 +59,18 @@ run {
     dependsOn copyNatives
     mainClassName = 'com.mycompany.app.App'
 }
+
+jar {
```
[the remark this answers is not in the document]
Comment (mbcoder) [reply build.gradle]: I've taken the task away

File: build.gradle [hunk 2]
```diff
@@ -20,7 +20,7 @@ plugins {
 }
 
 ext {
-    arcgisVersion = '100.11.2'
+    arcgisVersion = '100.12.0'
 }
 
 repositories {
```
[the remark this answers is not in the document]
Comment (mbcoder) [reply build.gradle:26]: Changed to mavenCentral and also updated grade wrapper to 7.1.1

File: build.gradle [hunk 3]
```diff
@@ -38,6 +38,7 @@ dependencies {
     implementation "com.esri.arcgisruntime:arcgis-java:$arcgisVersion"
     natives "com.esri.arcgisruntime:arcgis-java-jnilibs:$arcgisVersion"
     natives "com.esri.arcgisruntime:arcgis-java-resources:$arcgisVersion"
+    implementation "org.slf4j:slf4j-nop:1.7.3"
```
[the remark this answers is not in the document]
Comment (mbcoder) [reply build.gradle:41]: Oh yes!

File: gradlew.bat [hunk 4]
```diff
@@ -1,19 +1,20 @@
-@if "%DEBUG%" == "" @echo off
-
-@rem Copyright 2019 Esri
 @rem
-@rem Licensed under the Apache License, Version 2.0 (the "License"); you may not
-@rem use this file except in compliance with the License. You may obtain a copy
-@rem of the License at
+@rem Copyright 2015 the original author or authors.
```
Comment (Rachael-E) [inline gradlew.bat:2]:
```suggestion
@rem Copyright 2015 Esri
```

File: gradlew [hunk 5]
```diff
@@ -1,18 +1,20 @@
 #!/usr/bin/env sh
 
-#Copyright 2019 Esri
 #
-#Licensed under the Apache License, Version 2.0 (the "License"); you may not
-#use this file except in compliance with the License. You may obtain a copy
-#of the License at
+# Copyright 2015 the original author or authors.
```
Comment (Rachael-E) [inline gradlew:4]:
```suggestion
# Copyright 2015 Esri
```

**Review by Rachael-E — APPROVED** [review]
looks good, thanks @mbcoder ! looks like the update of the gradle wrappers has changed some copyright: it's probably ok as it is but have added comments where we might want to revert it back to Esri .

File: gradlew.bat (hunk 4 above)
Comment (mbcoder) [reply gradlew.bat]: fixed

File: gradlew (hunk 5 above)
Comment (mbcoder) [reply gradlew]: fixed

File: build.gradle [hunk 6]
```diff
@@ -38,6 +38,7 @@ dependencies {
     implementation "com.esri.arcgisruntime:arcgis-java:$arcgisVersion"
     natives "com.esri.arcgisruntime:arcgis-java-jnilibs:$arcgisVersion"
     natives "com.esri.arcgisruntime:arcgis-java-resources:$arcgisVersion"
+    runtimeOnly "org.slf4j:slf4j-nop:1.7.32"
```
Comment (jenmerritt) [inline build.gradle:41]: Samples have this comment above the slf4j line:
`// handle SLF4J http://www.slf4j.org/codes.html#StaticLoggerBinder`

Comment (Rachael-E) [inline build.gradle:41]: in the samples I've added a comment to explain to the user why this is here: might want to consider that here too

```
//   handle SLF4J http://www.slf4j.org/codes.html#StaticLoggerBinder

```

**Review by Rachael-E — APPROVED** [review]

**Review by jenmerritt — APPROVED** [review]
LGTM. Just one note of the comment Samples have for the slf4j line but entirely optional whether you want it in the starter project or not!

**Pull request merged**